src/common/obj_bencher.cc

```diff
@@ -376,7 +376,17 @@ int ObjBencher::fetch_bench_metadata(const std::string& metadata_file,
   } else {
     *op_size = *object_size;
   }
-  unsigned ops_per_object = *object_size / *op_size;
+
+  unsigned ops_per_object;
+
+  // in case *op_size came out wrong
+  if((*op_size == 0) || (*object_size < *op_size)) {
```

Comment:
please remove the parenthesis around `*op_size == 0` and `*object_size < *op_size`, they are unnecessary and distracting.

Reply:
This is outdated but I'll remove the parenthesis in `if ((*op_size > 0) && (*object_size > *op_size))`

Reply:
Parentheses removed